cookbook/prompt_management_openai_functions.ipynb

```diff
@@ -324,14 +324,19 @@
         "\"\"\""
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    summarize_story(STORY)
</pre><p>capture the output, otherwise on next execution it will be printed again</p><p> </p>

<pre class="ql-syntax" spellcheck="false">summary = summarize_story(STORY)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>